src/renderer/components/database-diagram.jsx

```diff
@@ -20,7 +18,7 @@ export default class DatabaseDiagram extends Component {
   constructor(props) {
     super(props);
     this.state = {};
-    this.graph = new joint.dia.Graph();
+    this.graph = new joint.dia.Graph({}, { cellNamespace });
```

Comment:
It was required to use this namespace definition for custom elements on version 3 of JointJS https://stackoverflow.com/a/57381434/1050818